doc/admin-guide/files/records.config.en.rst

```diff
@@ -416,6 +416,13 @@ Thread Variables
    This setting specifies the number of active client connections
    for use by :option:`traffic_ctl server restart --drain`.
```

Comment:
Hmm. you mean exposing an API to initiate the shutdown? @mlibbey also gave the same idea.